modules/ossm-uguide-bookinfo-example.adoc

```diff
@@ -0,0 +1,189 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_user_guide/ossm-traffic-manage.adoc
+
+[id="ossm-userguide-routing-bookinfo_{context}"]
+= Routing example using the bookinfo application
+
+The {ProductShortName} `bookinfo` sample application consists of four separate microservices, each with multiple versions. Three different versions of one of the microservices, called `reviews`, have been deployed and are running concurrently.
+
+.Prerequisites:
+
+* Deploy the `bookinfo` sample application to work with the following examples.
+
+.About this task
+
+To illustrate the problem this causes, access the `bookinfo` app's `/product page` in a browser and refresh several times.
+Sometimes, the book review output contains star ratings and other times it does not. Without an explicit default service version to route to, {ProductShortName} routes requests to all available versions one after the other.
+
+This tutorial helps you apply rules that route all traffic to `v1` (version 1) of the microservices. Later, you can apply a rule to route traffic based on the value of an HTTP request header.
+
+[id="ossm-userguide-routing-bookinfo-applying_{context}"]
+== Applying a virtual service
+
+To route to one version only, apply virtual services that set the default version for the micro-services. In the following example, the virtual service routes all traffic to `v1` of each micro-service
+
+1.  Run the following command to apply the virtual services:
+
+    ----
+    $ oc apply -f https://raw.githubusercontent.com/istio/istio/master/samples/bookinfo/networking/virtual-service-all-v1.yaml
```

Comment:
I think that instead of refer to Istio upstream (plus master branch) we should refer to maistra and the current branch (e.g. https://raw.githubusercontent.com/Maistra/istio/maistra-1.1/samples/bookinfo/networking/virtual-service-all-v1.yaml)